maas/plugins/nova_cloud_stats.py

```diff
@@ -106,6 +111,20 @@ def main(args):
     with print_output():
         parser = argparse.ArgumentParser(
             description='Check Nova hypervisor stats')
+        parser.add_argument('--cpu',
```

Comment:
These flag names could be more descriptive for people who test these scripts on the CLI.